composer.json

```
@@ -5,7 +5,8 @@
     "license": ["GPL-3.0-or-later"],
     "require": {
         "google/apiclient": "~2.2.0",
-        "neos/neos": "~3.0 || ~4.0 || dev-master"
+        "neos/neos": "~3.0 || ~4.0 || dev-master",
```

With these changes the compatibility with 3.x is gone. But I think it's fine to focus on the 4.3 LTS now.
```suggestion
        "neos/neos": "~4.3 || dev-master",
```